### ISSUE TITLE
Blog categories

### ISSUE THREAD

**Issue by KitWallace** [BristolOpenData/BristolOpenData.github.io#6]
Blog categories are mined from the words used in the blog post front matter category.  It would be sensible to define a list of categories to use.

**Comment by jennet** [comment]
Agree. This would makes sense to be a documentation issue rather than developer one. I'll see if I can sort out the labels here in the github issues list.

Who would like to volunteer to do this?

**Comment by KitWallace** [comment]
It's a development issue as well if the list is to be fixed and editable as a file rather than mined on the fly.

**Comment by jennet** [comment]
I think it would be best to keep it simple and just define the list of categories for people to add in the front matter (manually) when they are authoring a blog post.

Let's not overcomplicate things for now. Let's make it a docs issue for now and perhaps revisit at a later date for anything above and beyond that. Otherwise we'll get bogged down in scope growth at too early a stage.

**Comment by KitWallace** [comment]
My suggested categories:

News
Events
Data sets   - about datasets wanted, datasets found
Case Studies - case studies demonstrating the use of open data
Help Sheets - help sheets to carry out data tasks 

(the later entries don't have to be new material, it could be pointers to existing material)

**Comment by jennet** [comment]
Currently there are two main categories used for the site navigation: news and events
These are used for the sidebar links to extra posts

The blog page lists all posts regardless of category.

We should keep this issue open as I still think we need to define a list of categories for people to use in the front matter (e.g. your list suggested above)

During the next web development meeting we should all discuss the best way to navigate groups of posts on the site (and at that point I can create a new issue to support category pages - that lists out all posts separated by category)

**Comment by KitWallace** [comment]
It would be good to reflect categories in the colour of the entry in the blog index
